build.gradle

```diff
@@ -284,3 +284,44 @@ bootstrap.dependsOn installTestGems
 runIntegrationTests.shouldRunAfter tasks.getByPath(":logstash-core:test")
 check.dependsOn runIntegrationTests
 
+task runXPackUnitTests(type: Exec, dependsOn: installTestGems) {
```

Comment:
Ideally, we would respect `OSS=true` everywhere, and bail before loading commercial code (even if it is the tests, and even if a user explicitly asks us to run them).

My primary worry is that this task may eventually be accidentally made a dependency of another task, which could surprise a contributor who wanted to avoid running commercially-licensed code.